activestorage/test/models/blob_test.rb

```diff
@@ -80,6 +80,25 @@ class ActiveStorage::BlobTest < ActiveSupport::TestCase
     end
   end
 
+  test "record touched after analyze" do
+    user = User.create!(
+      name: "Nate",
+      avatar: {
+        content_type: "image/jpeg",
+        filename: "racecar.jpg",
+        io: file_fixture("racecar.jpg").open,
+      }
+    )
+
+    old_updated_at = user.reload.updated_at
+
+    # Analyze the blob
+    user.avatar.blob.analyze
+
+    # User should have been touched
+    assert_not_equal user.reload.updated_at, old_updated_at
```

Comment:
```suggestion
    assert_changes -> { user.reload.updated_at } do
       ...
```

I think you can use `assert_changes` here for clarity.

Reply:
Oh, nice!